src/components/SwagLandingCard.vue

```diff
@@ -24,14 +29,27 @@
     margin: 0;
     line-height: 133%;
   }
+.flex {
+    display: flex;
+    flex-direction: col;
+    align-items: center;
+  }
+.rounded-l-lg {
+    border-top-left-radius: 0.375rem;
+    border-bottom-left-radius: 0.375rem;
+  }
+  &:hover {
```

Comment:
See how to [handle states](https://tailwindcss.com/docs/hover-focus-and-other-states).